### ISSUE TITLE
preprocessed DTU error

### ISSUE THREAD

**Issue by YuhsiHu** [zezeaaa/MVPGS#3]
Hi, thank you for your great work!

When preprocessing DTU dataset, there are some errors in the code. 

1. The colmap path params in [run_colmap](https://github.com/zezeaaa/MVPGS/blob/master/utils/colmap2poses.py#L583) and colmap_executable [here](https://github.com/zezeaaa/MVPGS/blob/master/convert.py#L23). I can fix this with correct path.

2. The magick_command is missing in README, which results in failure:
```
sh: 1: magick: not found
```
What should I do to solve this? This is not mentioned at all.

Thank you for your time.

**Comment by zezeaaa** [comment]
Hi, thanks for your feedback! I have updated the code based on your issues.
- Our code runs colmap in [covert.py](https://github.com/zezeaaa/MVPGS/blob/2dbed03a2f9afc49428cf89d2892d77bb15eed61/convert.py#L75), so [run_colmap](https://github.com/zezeaaa/MVPGS/blob/2dbed03a2f9afc49428cf89d2892d77bb15eed61/utils/colmap2poses.py#L583) is not used during preprocessing. We can comment out that part of the code.
- ImageMagick is used to resize images; however, our code does not require it for resizing. We can set `--resize` to `False` in [dtu_colmap.py](https://github.com/zezeaaa/MVPGS/blob/2dbed03a2f9afc49428cf89d2892d77bb15eed61/dtu_colmap.py#L40). If you want to use ImageMagick, you can install it following [this link](https://imagemagick.org/script/download.php).

If you encounter any other issues, please let me know. Thank you!

**Comment by YuhsiHu** [comment]
Thank you for your quick reply! 

I found that MVPGS needs to first train on the scene, for example, T&T, which seems to differ from MVSGaussian can only train on DTU and then directly test on T&T (but can also fine-tune on it). Do you have any insight about why this happens?

**Comment by zezeaaa** [comment]
Yes, they are two different methods. MVPGS is still optimized per scene, similar to the original 3DGS or NeRF. While we utilize MVS to predict depth, we don't update the MVS model during optimization. We use this prior for initialization and to supervise unseen views. On the other hand, MVSGaussian is designed for generalization and the MVS module will be updated during training. This allows it to predict all the Gaussian parameters needed for rendering at once based on the trained model.
